sss.py

```diff
@@ -143,17 +131,45 @@ def load_wmv(path):
 
 
 def save_to_wmv(output_audio, path, sr):
-    sf.write(path, np.array(output_audio), sr, "PCM_24")
+    sf.write(path + '.wav', np.array(output_audio), sr, "PCM_24")
     print(f"File saved to {path}")
 
 
-if __name__ == "__main__":
-    args = get_args()
-    w_path = f"train/wage_matrices/{EXTRACTED_FEATURE}.npy"
+@click.command()
+@click.option('-t', '--type', default='VOCAL', help='type of the extraction',
+              type=click.Choice(['KARAOKE', 'BASS', 'DRUMS', 'VOCAL', 'FULL'], case_sensitive=False))
+@click.option('-m', '--method', default='NMF', help='extraction method',
+              type=click.Choice(['NMF'], case_sensitive=False))
+@click.option('-q', '--quality', default='NORMAL', help='choose extraction quality',
+              type=click.Choice(['FAST', 'NORMAL', 'HIGH'], case_sensitive=False))
+@click.option('-e', '--evaluation-data', default=None, nargs=2,
+              type=click.Tuple([click.Path(exists=True), click.Path(exists=True)]), help='extraction evaluation')
+@click.option('--reverse/--no-reverse', '-r/', default=False, help='reversed extraction')
+@click.option('-T', '--max-time', default=1000, type=click.IntRange(1,), help='maximum extraction time')
+@click.option('-I', '--max-iter', default=1000000000, type=click.IntRange(1,), help='maximum iterations number')
+@click.option('-o', '--output-file', default="results\\separated", type=click.Path(), help='output file location')
+@click.argument('input-file', type=click.Path(exists=True))
+def sss(type, method, quality, reverse, evaluation_data, max_time, max_iter, input_file, output_file):
+    print('type         ', type)
+    print('method       ', method)
+    print('quality      ', quality)
+    print('reverse      ', reverse)
+    print('evaluation   ', evaluation_data)
+    print('max_time     ', max_time)
+    print('max_iter     ', max_iter)
+    print('input_file   ', input_file)
+    print('output_file  ', output_file)
+
+    w_path = f"database\\train\\{EXTRACTED_FEATURE}.npy"
+    output_file += f'-{type}'
```

Comment:
Imo trzeba będzie rozdzielić funkcjonalność programu, odpalenie konsolowe oraz GUI, ale to w osobnych gałęziach